lib/intro.stub

```diff
@@ -1,5 +1,5 @@
 /*! axe v<%= pkg.version %>
- * Copyright (c) <%= grunt.template.today("yyyy") %> Deque Systems, Inc.
+ * Copyright (c) 2015 Deque Systems, Inc.
```

Comment:
Lets do this, so we have the date of the last update in here. Makes the files not seem old.

```suggestion
 * Copyright (c) 2015 - <%= grunt.template.today("yyyy") %> Deque Systems, Inc.
```